### DISCUSSION TITLE
download

### DISCUSSION THREAD

**Discussion by DaggerSepulcher** [ruffle-rs/ruffle#3343] [Q&A] [answered]
if i were to want to download a swf from a site thats playing on ruffle, how would i go about that?     

window.RufflePlayer = window.RufflePlayer || {};

        $(document).ready(function() {
            let ruffle = window.RufflePlayer.newest();
            let player = ruffle.createPlayer();
            $('#container').append(player);
            player.load("/data/animation

but i dont see a way to get to that directory

**Comment by nagatsu-chan** [comment, marked as answer]
Konnichiwaaa xD xD

i see u havingu a toughu time!1!11!!!!!!1! xD it ok don'tsu worryu though!1!11!!!!!!1! xD 

justs open developer toolsu ,and, go tah da networku tab, ,and, search for any files thats end in .swf!1!11!!!!!!1! xD make sure u have da networku tab open before u try searchingu ~ then just rightsie clickie on da file you wantsu tah download, ,and, click open in new tab!1!11!!!!!!1! xD 

Hwope this help senpai Miles-chan UwU

**Reply by DaggerSepulcher** [reply]
ありがとうございました >^.^<

**Reply by DaggerSepulcher** [reply]
really? cause thats exactly where it was.
